modules/ROOT/pages/hybrid-schedule-mgmt.adoc

```diff
@@ -32,14 +32,14 @@ Runtime Manager lists the name of each Scheduler component as `polling://{flow_n
 
 Schedule management is useful in the following contexts:
```

Comment:
Missing "Schedule"

Reply:
No, it's ok. Because it is the schedule of the scheduler. Right?

Reply:
@fpiacentini08 - I'm not sure what you're asking. Should it be Scheduler management?

Reply:
I think it is ok